### PR TITLE
name the fields in longer constructors

### PR DESCRIPTION
**Pull request by exaexa**
to avoid breaking everything everytime a field is added. :]

Also soaked up one related fix of the Event

### CODE REVIEW

**Comment by giordano** [comment]
Ehm... #168 :stuck_out_tongue:

File: src/readsbml.jl [hunk 1]
```diff
@@ -458,7 +458,7 @@ function extract_model(mdl::VPtr)::SBML.Model
 
     initial_assignments = Dict{String,Math}()
     num_ias = ccall(sbml(:Model_getNumInitialAssignments), Cuint, (VPtr,), mdl)
-    for n in 0:(num_ias - 1)
+    for n = 0:(num_ias-1)
```
Comment (giordano) [inline src/readsbml.jl:461]: Side note, the fact that you can do for loops with the `for var = range` syntax confuses some new users.  I wish that syntax wasn't allowed at all, and I'm not the only one.

Comment (exaexa) [reply src/readsbml.jl:461]: ha, this is actually enforced by juliaformatter. Should I complain there?

Comment (giordano) [reply src/readsbml.jl:461]: Ugh.  Yes, please, if you're willing to do it :smile:

Comment (exaexa) [reply src/readsbml.jl:461]: the point is that `in` collides nicely with the `in` operator in a way that is IMO even worse. Why's there even the separator when the parse is unambiguous anyway? :D

**Pull request closed without merging**